src/styles/StyleUtils.ts

```diff
@@ -908,11 +798,8 @@ function getHorizontalStackedAvatarStyle(index, overlapSize) {
 
 /**
  * Get computed avatar styles of '+1' overlay based on size
- * @param {Object} oneAvatarSize
- * @param {Numer} oneAvatarBorderWidth
- * @returns {Object}
  */
-function getHorizontalStackedOverlayAvatarStyle(oneAvatarSize, oneAvatarBorderWidth) {
+function getHorizontalStackedOverlayAvatarStyle(oneAvatarSize: {width: number}, oneAvatarBorderWidth: number): ViewStyle | CSSProperties {
```

Comment:
Let's extract `{width: number}` to a type

Reply:
Done!